.gitignore

```diff
@@ -1,3 +1,3 @@
-!.gitignore
+#.gitignore
 node_modules/
 test.md
```

Comment:
- [ ] Good job adding the ```test.md``` file to the .gitignore, however, you need to remove it from your branch. you can remove ```test.md``` using [this tutorial](https://fedingo.com/how-to-remove-git-ignore-files-from-git-repository/)